src/main/java/net/vulkanmod/vulkan/Drawer.java

```diff
@@ -181,7 +181,7 @@ public void initiateRenderPass() {
             renderPassInfo.renderArea(renderArea);
 
             VkClearValue.Buffer clearValues = VkClearValue.callocStack(2, stack);
-            clearValues.get(0).color().float32(stack.floats(0.0f, 0.0f, 0.0f, 1.0f));
```

Comment:
Color can be provided to renderpass directly instead of just clear attachments

Reply:
Apparently your not supposed to use too many clear colors (15-30) at least on Nvidia, which disabled color compression apparently and wastes bandwidth, but in practise saw virtually no difference to Perf what so ever even when forcibly disabling all clears.